docs/install/local/README.md

```diff
@@ -257,6 +257,9 @@ fileStore:
 
 ## Upgrade
 
+If upgrading from 1.x.y to 2.x.y then you may need to upgrade from NodeJS v16 to NodeJS v18.
```

Comment:
Is this referring to FlowFuse or localfs Ben?

Reply:
I don't understand the question Steve. This is on the FlowFuse localfs install page.

Reply:
Aaaarrrrrgggghhhh, complete brain f*** - forgot localfs was part of FlowFuse 🤦